packages/ember-metal/lib/transaction.js

```diff
@@ -0,0 +1,75 @@
+import { meta as metaFor } from './meta';
+import { has } from 'require';
+import { deprecate, runInDebug } from 'ember-metal/debug';
+import isEnabled from 'ember-metal/features';
+
+let runInTransaction, didRender, assertNotRendered;
+
+if (isEnabled('ember-glimmer-allow-two-way-reflush')) {
+  let hasGlimmer = has('glimmer-reference');
+  let counter = 0;
+  let inTransaction = false;
```

Comment:
I think this is ok because we reset this after the transaction. You currently cannot have concurrent renders so this should just work.